app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,15 @@
+class OrdersController < ApplicationController
+  include ActionView::Helpers::TextHelper
+
+  def show
+
+  end
+
+  def create
+    flash[:notice] = "Successfully submitted your order totaling $#{@basket.total_cost}"
+    @basket.checkout
```

Comment:
Any thoughts on using this to dump the basket into an order in the model?

Reply:
I like the PORO having the responsibility of creating the order. It moves the responsibility out of the controller.